src/test/java/org/swisspush/reststorage/lua/AbstractLuaScriptTest.java

```diff
@@ -35,6 +36,7 @@
     final static String expirableSet = "rest-storage:expirable";
     final static String prefixDeltaResources = "delta:resources";
     final static String prefixDeltaEtags = "delta:etags";
+    final static String prefixLock = "redis-storage:lock";
```

Comment:
I would suggest to set the default value of lockPrefix to **rest-storage:locks** since there will be multiple locked resources below this path.